ruby-training/trainees/shareni/test_katas/04/converter/converter.rb

```diff
@@ -0,0 +1,10 @@
+class Converter
+    def convert_number_to_reverse_array(original_number)
+       convert_number_to_array(original_number).reverse 
+    end
+
+    def convert_number_to_array(original_number)
+        numbers = original_number.to_s.split('')
```

Comment:
Maybe call this variable something like `individual_digits`, as numbers can have multiple digits and you're explicitly separating each digit into it's char value.